.changeset/olive-trains-walk.md

```diff
@@ -0,0 +1,11 @@
+---
+"@ponder/core": patch
+---
+
+**BREAKING** Migrated to [viem](https://viem.sh). Notes:
+
+Ponder projects must now use **Node 18** or a fetch polyfill (see [viem docs](https://viem.sh/docs/compatibility.html)).
```

Comment:
Alternatively `node --experimental-fetch`

Reply:
Oh, interesting. I suppose users could change their package.json like this (just tested, works on Node 16). Pretty ugly though!

```json
// before
{
  "scripts": {
    "dev": "ponder dev",
  },
}

// after
{
  "scripts": {
    "dev": "node --experimental-fetch node_modules/@ponder/core/dist/bin/ponder.js dev",
  },
}
```